gulp/sass.js

```diff
@@ -70,8 +69,9 @@ module.exports = (gulp, plugins, blueprint) => {
                     );
                 },
             }),
-            autoprefixer(config.autoprefixer),
-        ];
+        ] : [];
+        // always run autoprefixer
+        postcssPlugins.push(autoprefixer(config.autoprefixer));
```

Comment:
Alternatively, this could be in each of the arrays in the ternary above. That way, if we wanted to do something AFTER autoprefixer in either case, it would be easier to modify.

Not a big deal though. Your call.